operators/Makefile

```diff
@@ -301,8 +309,11 @@ ci: dep-vendor-only check-fmt generate check-local-changes unit integration e2e-
 
 # Run e2e tests in a dedicated gke cluster,
 # that we delete if everything went fine
+# TODO: It needs n1-standard-8 since 2 pods may be scheduled on the same machine.
+# We should be able to remove this when K8S min. version will be 1.12 and we can use
+# anti-affinity and storage class with volumeBindingMode=WaitForFirstConsumer.
```

Comment:
is the use of `n1-standard-8 ` related to the PSP addition?

Reply:
no, I did several tests and sometimes it fails just because 2 pods are scheduled on the same node.

Reply:
I'm not sure about the TODO comment, it looks to me like a TODO that will stay there forever:

* we will probably keep on testing 1.12 for a long time
* many of our e2e tests may not bother with anti-affinity and compatible storage classes

I would strip it to just `Let's use n1-standard-8 machine to have enough room for multiple pods on a single node`.